src/modules/Workspaces/WorkspacesEditor/ViewModels/MainViewModel.cs

```diff
@@ -601,5 +603,37 @@ private void SendDeleteTelemetryEvent()
             telemetryEvent.Successful = true;
             PowerToysTelemetry.Log.WriteEvent(telemetryEvent);
         }
+
+        internal static bool IsPwaApp(Models.Application application)
+        {
+            if (application.IsEdge())
+            {
+                return true;
+            }
+            else if (application.IsChrome())
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        internal static string GetPwaIconFilename(Models.Application application)
+        {
+            if (application.IsEdge())
+            {
+                return PwaHelper.GetEdgeAppIconFile(application.PwaAppId);
+            }
+            else if (application.IsChrome())
+            {
+                return PwaHelper.GetChromeAppIconFile(application.PwaAppId);
+            }
+            else
+            {
+                return string.Empty;
+            }
+        }
```

Comment:
This method looks similar in Editor and LauncherUI, so it makes sense to have it in a common library.